scrape/scrape.go

```diff
@@ -805,11 +812,17 @@ func (c *scrapeCache) get(met string) (*cacheEntry, bool) {
 	return e, true
 }
 
-func (c *scrapeCache) addRef(met string, ref uint64, lset labels.Labels, hash uint64) {
+func (c *scrapeCache) addRef(met string, ref uint64, lset labels.Labels, meta storage.Metadata, hash uint64) {
```

Comment:
Passing structs around is inefficient, use a pointer.

Reply:
Sounds good, will do.